libs/portal/features/profile/ui/header.tsx

```
@@ -0,0 +1,49 @@
+import { UserProfileStatus } from '@cuhacking/portal/types/user';
+import {
+  Avatar,
+  AvatarFallback,
+  AvatarImage,
+} from '@cuhacking/shared/ui/avatar';
+
+interface QuestionHeaderProps {
+  status: UserProfileStatus;
```

ESLint

[error] Unexpected separator (;).

(style/member-delimiter-style)